packages/mobile/src/store/offline-downloads/sagas.ts

```diff
@@ -216,6 +217,7 @@ export function* watchSetReachable() {
 }
 
 export function* handleSetUnreachable() {
+  // Why do we do this?
   yield* call(setPlayCounterWorker, blockedPlayCounterWorker)
```

Comment:
I think you won't need this if the queue is properly stopped while offline.
This was a hack to keep a playCounterWorker attached to the queue that won't process any tracks. The blockedPlayCounterWorker has concurrency 0 so it won't pick up tracks.